docs/computed-fields.md

```diff
@@ -0,0 +1,65 @@
+# Computed Fields
+
+A card’s fields can link to or contain other cards. A special
+In addition to linking to or containing card fields, a card can have a “computed” field. Its value will be determined by the output of the `computeVia` function and update when its constituent fields changes.
+
+```typescript
+export class Person extends Card {
+  @field firstName = contains(StringCard);
+  @field lastName = contains(StringCard);
+
+  @field fullName = contains(StringCard, {
+    computeVia: function (this: Person) {
+      return `${this.firstName ?? ''} ${this.lastName ?? ''}`;
+    }
+  });
+  …
+```
+
+You can also specify the name of the computation function as a string instead of inlining it:
+
+```typescript
+@field fullName = contains(StringCard, { computeVia: 'getFullName' });
+
+getFullName() {
```

Comment:
I assume this needs to be a method name in the same class?

Reply:
It’s true, I’ve updated to clarify that, thanks.